src/integTest/groovy/net/wasdev/wlp/gradle/plugins/AbstractIntegrationTest.groovy

```diff
@@ -50,8 +43,8 @@ abstract class AbstractIntegrationTest {
             }
         }
     }
-
-    protected static File createTestProject(File parent, File sourceDir, String buildFilename) {
+    
+    protected static File createTestProject(File parent, File sourceDir) {
```

Comment:
I think a better fix may be to call `renameBuildFile` within 'createTestProject' to make configuring a test case simpler